tests/integration/test_sessions.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import json
+from ert_storage.client.session import Session
+
+HTTP_OK_STATUS = 200
```

Comment:
Nitpick: use `from fastapi import status` which provides these same constants, but in a centralised location.

Reply:
Did consider that but I have a general aversion to adding dependency to tests just for consts, Happy to change it over though